lambdas/postgres-migration-count-tool/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path');
```

Comment:
This looks like it shouldn't be being pulled forward.

Reply:
https://github.com/nasa/cumulus/pull/3225/commits/f6ec3a42afde116a290c05556fed107bb0c2a952